industry_benchmarks/utils/plan_rbfe_network.py

```diff
@@ -256,7 +263,24 @@ def run_inputs(ligands, pdb, cofactors, output):
                 protocol=rbfe_protocol,
                 name=name
             )
-            transformation.dump(output / f"{transformation.name}.json")
+            transformations.append(transformation)
+
+    # Create the alchemical network and write it to disk
+    alchemical_network = openfe.AlchemicalNetwork(transformations)
+    alchemical_network_json_fp = output / "alchemical_network.json"
```

Comment:
Is it always going to be `"alchemical_network.json"`

Reply:
yes I believe so